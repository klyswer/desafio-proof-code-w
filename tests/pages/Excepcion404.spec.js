import React from 'react';
/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import Exc404 from '../../src/pages/Exc404';

describe('<Exc404 />', () => {

  test('Primer SnapShot: <Exc404 />', () => {
    const wrapper = shallow( <Exc404 />);
    expect(wrapper).toMatchSnapshot();
  });
  
});
