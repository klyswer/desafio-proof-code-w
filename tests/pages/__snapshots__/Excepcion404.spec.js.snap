// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Exc404 /> Primer SnapShot: <Exc404 /> 1`] = `
<Fragment>
  <h1>
    Not found 404
  </h1>
</Fragment>
`;
